<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HW1352" failures="0" tests="1" name="playwright.tests.downlaodFile" time="19.980" errors="1" timestamp="12 Sept 2025 13:35:48 GMT" skipped="0">
  <testcase classname="playwright.tests.downlaodFile" name="handleDownloadFile" time="19.980">
    <error message="Input length = 3" type="java.nio.charset.MalformedInputException">
      <![CDATA[java.nio.charset.MalformedInputException: Input length = 3
at java.base/java.lang.String.throwMalformed(String.java:1279)
at java.base/java.lang.String.decodeUTF8_UTF16(String.java:1166)
at java.base/java.lang.String.newStringUTF8NoRepl(String.java:759)
at java.base/java.lang.String.newStringNoRepl1(String.java:790)
at java.base/java.lang.String.newStringNoRepl(String.java:772)
at java.base/java.lang.System$2.newStringNoRepl(System.java:2480)
at java.base/java.nio.file.Files.readString(Files.java:3362)
at java.base/java.nio.file.Files.readString(Files.java:3318)
at playwright.tests.downlaodFile.handleDownloadFile(downlaodFile.java:56)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- handleDownloadFile -->
</testsuite> <!-- playwright.tests.downlaodFile -->
